static/js/containers/pages/DashboardPage.js

```diff
@@ -54,6 +57,26 @@ export class DashboardPage extends React.Component<Props, State> {
   }
 
   componentDidMount() {
+    if (
+      typeof CSOURCE_PAYLOAD === "object" &&
+      CSOURCE_PAYLOAD &&
+      SETTINGS.gaTrackingID
+    ) {
+      try {
+        dataLayer.push({
+          event:            "purchase",
+          transactionId:    CSOURCE_PAYLOAD.transaction_id,
+          transactionTotal: CSOURCE_PAYLOAD.transaction_total,
+          productType:      CSOURCE_PAYLOAD.product_type,
+          coursewareId:     CSOURCE_PAYLOAD.courseware_id,
+          referenceNumber:  CSOURCE_PAYLOAD.reference_number
+        })
+      } catch (error) {
+        // eslint-disable-line no-empty
```

Comment:
What error are we trying to silence here? It seems like the above `dataLayer.push(...)` code should always work.

Reply:
Removed, seems to be a leftover from a previous implementation that was overlooked.